### PR TITLE
docs: fix wording in auto-updater.md

### PR DESCRIPTION
**Pull request by nikwen**
#### Description of Change

Simple wording fix in autoUpdater docs

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation, tutorials, templates and examples are changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/main/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/main/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

File: docs/api/auto-updater.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Emitted when there is an error while updating.
 
 ### Event: 'checking-for-update'
 
-Emitted when checking if an update has started.
+Emitted when checking if an update is available has started.
```
Comment (VerteDinde) [inline docs/api/auto-updater.md:71]: Not trying to bikeshed too much, but I think this wording is still a little confusing. What do you think of?:
```suggestion
Emitted when checking for an available update has started.
```

I like clarifying the check is for available updates and not for updating itself though 🙂

Comment (nikwen) [reply docs/api/auto-updater.md:71]: Works for me! I see what you mean. Thanks for the comment!

**Review by VerteDinde — APPROVED** [review]
Thanks for the fix!

**Comment by nikwen** [comment]
Thanks for merging, @VerteDinde! :raised_hands:

**Pull request merged**